docs/src/api.md

```diff
@@ -52,19 +52,40 @@ nextid
 random_position
 ```
 
+### Removing agents
+```@docs
+kill_agent!
+genocide!
+sample!
+```
+
 ### Moving agents
 ```@docs
 move_agent!
 walk!
 ```
 
-### Removing agents
+## Path-finding
+In addition to the direct movement functions listed above, [`GridSpace`](@ref) has the additional benefit of path planning.
 ```@docs
-kill_agent!
-genocide!
-sample!
+AStar
+set_target!
+is_stationary
+walkmap
+heightmap
```

Comment:
There is no reason for the functions `walkmap, heightmap` to exist, and even less so to be exported as part of the public API.

Reply:
Why not? You say a few comments above that the maps can/should be mutable by the user, how would they best do that&mdash;especially if we hide the `Pathfinder` struct completely from them?

Reply:
Okay, I see. So these functions can then used to mutate these two data in place, as I suggested in another comment. Okay then we have to document this and we are good to go.

Reply:
They should be renamed to `walkmap!` and `heightmap!` to make the mutation visible.

Reply:
They don't have to. You can do `walkmap(model) .= rand(10, 10)` and `walkmap(model)[1, 2] = 3`.

Reply:
it just has to be documented.